tasks/gulp/compile-scss.js

```diff
@@ -27,7 +27,7 @@ gulp.task('scss:compile-all-govuk-and-hmrc', () => gulp.src(`${configPaths.src}a
   // minify css add vendor prefixes and normalize to compiled css
   .pipe(postcss([
     autoprefixer,
-    cssnano,
+    // cssnano,
```

Comment:
@goodvibes we need to uncomment this so that the css gets minified

Reply:
Ohhhhhhh